jwst/photom/photom_step.py

```diff
@@ -26,6 +26,8 @@ def process(self, input_file):
             self.log.debug('Input is a CubeModel for a multiple integ file.')
         elif isinstance(dm, datamodels.ImageModel):  # standard product: 2D array
             self.log.debug('Input is an ImageModel.')
+        elif isinstance(dm, datamodels.IFUImageModel):  # standard product: 2D array
```

Comment:
Same comment as earlier about trying to find a simpler way to just print out the datamodel type.